packages/cordova/package.json

```diff
@@ -77,6 +77,10 @@
     },
     "scripts": {
         "build": "webpack",
-        "start": "npm run build && cordova run"
+        "start": "npm run build && cordova run",
+        "prepare": "npm run build && cordova prepare",
```

Comment:
This runs on `npm install` for me. is that intentional?

Reply:
I needed to run it manually, but let me try removing this and running on CI, to see if it works.

Reply:
Yep, without it, CI fails: https://github.com/padloc/padloc/runs/4625826891?check_suite_focus=true

Reply:
Let's keep it in then 👍